main.css

```diff
@@ -1,3 +1,13 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap');
```

Comment:
- [ ] You've forgotten to import the font-weight 600 for the Poppins Family.  Your mobile menu items need it, and that is why they are not rendering well (you declare 600 but the actual render is 700).

Just correct your import like this to include the weight 600:

```CSS
@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap');
```

That way your fonts will be really simmilar! 💪